### PR TITLE
fix: add compression icon to .tXX files

### PR DESCRIPTION
**Pull request by jim4067**
#930 

Compression icons for .tXX files

### CODE REVIEW

File: src/output/icons.rs [hunk 1]
```diff
@@ -302,6 +302,9 @@ pub fn icon_for_file(file: &File<'_>) -> char {
             "svg"           => '\u{f1c5}', // 
             "swift"         => '\u{e755}', // 
             "tar"           => '\u{f410}', // 
+            "tar.gz"        => '\u{f410}', // 
+            "tar.lz"        => '\u{f410}', // 
+            "tar.xz"        => '\u{f410}', // 
```
Comment (izhakjakov) [inline src/output/icons.rs:307]: I think "tgz", "tlz", "txz" should be added as well :)

Comment (ariasuni) [reply src/output/icons.rs:307]: They should added *instead*. Here, only the part after the last dot is matched against, and gz, lz and xz are already in the list.

File: src/info/filetype.rs [hunk 2]
```diff
@@ -80,7 +80,8 @@ impl FileExtensions {
         file.extension_is_one_of( &[
             "zip", "tar", "Z", "z", "gz", "bz2", "a", "ar", "7z",
             "iso", "dmg", "tc", "rar", "par", "tgz", "xz", "txz",
-            "lz", "tlz", "lzma", "deb", "rpm", "zst",
+            "lz", "tlz", "lzma", "deb", "rpm", "zst", "tar.gz",
+            "tar.xz", "tar.lz",
```
Comment (ariasuni) [inline src/info/filetype.rs:84]: The extension extracted by exa is only the part after the last dot, and gz, lz and xz are already in the list. So you should remove this change.

**Review by ariasuni — CHANGES_REQUESTED** [review]

File: src/info/filetype.rs (hunk 2 above)
Comment (jim4067) [reply src/info/filetype.rs:84]: Noted

File: src/output/icons.rs (hunk 1 above)
Comment (izhakjakov) [reply src/output/icons.rs]: Hi @ariasuni I think @jim4067 forgot to mention that he already addressed your comment in commit 437cd5b7f40ba1b704de211300b2eed42c93e71b

**Comment by izhakjakov** [comment]
Are there any comments on changes made to commit 437cd5b7f40ba1b704de211300b2eed42c93e71b ?

**Comment by codeswhite** [comment]
Missing `.lz4`

**Comment by jim4067** [comment]
> Missing `.lz4`

I checked and  the last PR made was for frequently used file types. @ariasuni what do you think?

**Comment by ariasuni** [comment]
oops I merged the other PR, but I’m pretty sure `.lz4` should be supported, especially since `.lz` is already supported. If you could create an new PR adding it in both places (`src/output/icons.rs` and `src/info/filetype.rs`), it would be perfect!

**Comment by jim4067** [comment]
Okay. I'll make a PR for it too 👍🏾

**Pull request closed without merging**